docs/api/networkinterfaces/WifiInterface.md

```diff
@@ -56,11 +56,11 @@ Please note that settings should match the security mode from the access point.
 
 ### Wi-Fi example
 
-Here is an example of an HTTP client program. The program brings up an ESP8266 as the underlying network interface, and uses it to perform an HTTP transaction over a TCPSocket. There are multiple [Wi-Fi components](https://os.mbed.com/components/cat/wifi/) that implement the WiFiInterface class. The example below uses the [ESP8266Interface](https://github.com/armmbed/esp8266-driver) and [OdinWiFiInterface](https://github.com/u-blox/ublox-odin-w2-drivers-docs-mbed-5).
+This program scans for Wi-Fi access points in the area. There are multiple [Wi-Fi components](https://os.mbed.com/components/cat/wifi/) that implement the WiFiInterface class. The example below uses the [ESP8266Interface](https://github.com/armmbed/esp8266-driver) and [OdinWiFiInterface](https://github.com/u-blox/ublox-odin-w2-drivers-docs-mbed-5).
 
 The ESP8266Interface uses AT commands over serial interface to connect to an external Wi-Fi device. The OdinWiFiInterface provides an Ethernet-like driver to the Mbed OS network stack. The network stack uses the driver to connect to Wi-Fi:
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-wifi)](https://github.com/ARMmbed/mbed-os-example-wifi/blob/mbed-os-5.10.0/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-wifi)](https://github.com/ARMmbed/mbed-os-example-wifi/blob/master/main.cpp)
```

Comment:
Is there a 5.11 branch or tag for this example?

Reply:
Does it need to be specifically to 5.11? Master should always point to the latest tag, Seppo says.

Reply:
Because we have versioned code, we must have versioned docs (and therefore, versioned examples). Our development branch can point to master, but our live sites should point to versioned examples. If someone's using Mbed OS 5.8, and we include an example from Mbed OS 5.11, which uses features not in 5.8, it'll be confusing.

Reply:
There are 4 different tags for 5.11 for this example, 1 for each RC, but not for the release

Reply:
Do we have some automation for updating the tags and links when mbed-os release is done, or is this all manual work?

Reply:
It's all manual work for now, but we've asked the teams revising the examples process to change that.